.github/workflows/maven-automatic.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Maven Automatic Package
```

Comment:
We should come up with more descriptive names.

Maybe "Automatic Build & Package"
And the other job is "Manual Build, Package & Release"